src/hotspot/cpu/x86/x86.ad

```diff
@@ -9018,6 +9019,63 @@ instruct vmask_gen_imm(kReg dst, immL len, rRegL temp) %{
   ins_pipe( pipe_slow );
 %}
 
+instruct loop_vmask_gen_small_trip(kReg dst, rRegI from, rRegI to, rRegI tmp1, rRegL tmp2) %{
```

Comment:
We probably need to mark rflags being killed for this. See https://github.com/openjdk/jdk/pull/16680